pygmt/tests/test_accessor.py

```diff
@@ -105,24 +135,24 @@ def test_accessor_grid_source_file_not_exist():
     Check that the accessor fallbacks to the default registration and gtype when the
     grid source file (i.e., grid.encoding["source"]) doesn't exist.
     """
-    # Load the 05m earth relief grid, which is stored as tiles
+    # Load the 05m earth relief grid, which is stored as tiles.
```

Comment:
Do comments need to end with a full stop now?

Reply:
That's just my preference, since it's a complete sentence.

Reply:
Ah ok, I was afraid that you were using some experimental ruff linter :laughing:!